math-magic/components/Calculator.js

```diff
@@ -0,0 +1,11 @@
+import './css/Calculator.css';
+import Child from './CalculatorChild';
+
+const Component = () => (
+  <div className="cal-contianer">
+    <div className="diplay">0</div>
+    <Child />
+  </div>
+);
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍